prod_environments.json

```diff
@@ -5,6 +5,7 @@
     "DOWNLOAD_CDN_URL": "https://svc90.main.px.t-online.de",
     "VERIFICATION_CDN_URL": "https://verification.coronawarn.app",
     "DATA_DONATION_CDN_URL": "https://data.coronawarn.app",
+    "LOG_UPLOAD_SERVER_URL": "https://placeholder",
```

Comment:
Once merged, please ping the team to remind them about the need to update the environment.json